src/Nethermind/Nethermind.Synchronization/Blocks/FastSyncFeed.cs

```diff
@@ -25,18 +25,7 @@ public FastSyncFeed(ISyncConfig syncConfig)
 
         private DownloaderOptions BuildOptions()
         {
-            DownloaderOptions options = DownloaderOptions.MoveToMain;
-            if (_syncConfig.DownloadReceiptsInFastSync)
-            {
-                options |= DownloaderOptions.WithReceipts;
-            }
-
-            if (_syncConfig.DownloadBodiesInFastSync)
-            {
-                options |= DownloaderOptions.WithBodies;
-            }
-
-            return options;
+            return DownloaderOptions.Insert | DownloaderOptions.WithReceipts;
```

Comment:
Why does it simplify?

Reply:
The receipt one, it does not make much of a difference, but it does make it less confusing why if you start a node without old block and receipts, and then restart with old block and receipts it would have a gap after the sync pivot.